src/citrine/resources/predictor_evaluation.py

```diff
@@ -0,0 +1,219 @@
+                                        per_page=per_page)
+
+    def list_all(self,
+                 *,
+                 per_page: int = 100,
+                 predictor_id: Optional[UUID] = None,
+                 predictor_version: Optional[Union[int, str]] = None
+                 ) -> Iterable[PredictorEvaluation]:
+        """List all predictor evaluations."""
+        return self._list_base(per_page=per_page,
+                               predictor_id=predictor_id,
+                               predictor_version=predictor_version)
+
+    def list(self,
+             *,
+             per_page: int = 100,
+             predictor_id: Optional[UUID] = None,
+             predictor_version: Optional[Union[int, str]] = None
+             ) -> Iterable[PredictorEvaluation]:
+        """List non-archived predictor evaluations."""
+        return self._list_base(per_page=per_page,
+                               predictor_id=predictor_id,
+                               predictor_version=predictor_version,
+                               archived=False)
+
+    def list_archived(self,
+                      *,
+                      per_page: int = 100,
+                      predictor_id: Optional[UUID] = None,
+                      predictor_version: Optional[Union[int, str]] = None
+                      ) -> Iterable[PredictorEvaluation]:
+        """List archived predictor evaluations."""
+        return self._list_base(per_page=per_page,
+                               predictor_id=predictor_id,
+                               predictor_version=predictor_version,
+                               archived=True)
+
+    def archive(self, uid: Union[UUID, str]):
+        """Archive an evaluation."""
+        url = self._get_path(uid, action="archive")
+        result = self.session.put_resource(url, {}, version=self._api_version)
+        return self.build(result)
+
+    def restore(self, uid: Union[UUID, str]):
+        """Restore an archived evaluation."""
+        url = self._get_path(uid, action="restore")
+        result = self.session.put_resource(url, {}, version=self._api_version)
+        return self.build(result)
+
+    def default_from_config(self, config: GraphPredictor) -> List[PredictorEvaluator]:
+        """Retrieve the default evaluators for an arbitrary (but valid) predictor config.
+
+        See :func:`~citrine.resources.PredictorEvaluationCollection.default_evaluators` for details
+        on the resulting evaluators.
+        """
+        path = self._get_path(action="default-from-config")
+        payload = config.dump()["instance"]
+        result = self.session.post_resource(path, json=payload, version=self._api_version)
+        return PredictorEvaluatorsResponse.build(result).evaluators
+
+    def default(self,
+                *,
+                predictor_id: Union[UUID, str],
+                predictor_version: Union[int, str] = LATEST_PRED_VER
```

Comment:
The predictor version defaults to `latest` for consistency with `trigger`, which requires a trained predictor.